src/manifest.chrome.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Cyber Citizenship Coefficient",
```

Comment:
Change the description

Reply:
What would you like me to change the description to exactly?

Reply:
`Plugin used to report poor web development behavior and to avoid bad sites`?

Reply:
Ok fixed.